tutorial/20_recipes/013_wilcoxon_pruner.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _wilcoxon_pruner:
+
+Early-stopping independent evaluations by WilcoxonPruner
+============================================================
+
+This tutorial showcases Optuna's WilcoxonPruner.
```

Comment:
WilcoxonPruner can be made a link.

Reply:
I added link of the latest document because stable document do not have WilcoxonPruner yet.
https://optuna.readthedocs.io/en/latest/reference/generated/optuna.pruners.WilcoxonPruner.html